src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -95,7 +102,7 @@ const SignedChatScreen = () => {
         />
       ) : null}
       <FlatList
-        contentContainerStyle={styles.contentContainerStyle}
+        contentContainerStyle={styles.flatlistContainer}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you've renamed the style property `contentContainerStyle` to `flatlistContainer`. Make sure this new style property exists and is defined correctly in your styles file.



<!-- This is an auto-generated comment by OSS CodeRabbit -->